install_and_start_redis

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+REDIS_VERSION="6.2.4"
+pwd=$PWD
+TEST_DIR=$PWD/test_dependencies
+REDIS_SERVER_SCRIPT=$TEST_DIR/redis-$REDIS_VERSION/src/redis-server
+
+if [ $(ps aux | grep -v grep | grep -c $REDIS_SERVER_SCRIPT) -gt 0 ]
```

Comment:
The usual way to manage servers is to track their PID in a file.  That's far less fragile than this, which might end up thinking that a different server instance is running.